src/components/ui/search-bar/search-bar-renderer.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import SearchBarRenderer from './search-bar-renderer';
+
+configure({ adapter: new Adapter() });
```

Comment:
is this set up needed for this test?

Reply:
not sure. this file was a 1-to-1 migration. will check